02-Reproject-Raster-In-R.Rmd

```diff
@@ -42,6 +43,12 @@ After completing this activity, you will:
 
 * Be able to reproject a raster in R
 
+###Challenge Code
```

Comment:
Rather than having this on EVERY LESSON. i wonder if we create ONE PAGE that overviews how our lessons work. Then we can link to it on each page that has this feature.